### ISSUE TITLE
[Accessibility: ACC-264.1] Multi Combobox: Information not read

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12571]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Selected entered value within all input fields should be read.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All versions.

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Multi Combobox](https://sap.github.io/fundamental-ngx/#/core/multi-combobox)
Try to read selected values in multi combobox with screen reader. Value is not read.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/3cf240c5-45d8-4486-be72-8b30a3a4921c">

#### Please provide stackblitz example(s).
You can refer ngx example, it is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
You can try ngx examples, issue is reproducible there.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 I tried the same scenario on our [doc](https://fundamental-ngx.netlify.app/#/platform/variant-management) and everything seems to be good.
<img width="739" alt="Screenshot 2024-10-21 at 14 45 18" src="https://github.com/user-attachments/assets/da70ca10-b904-40f5-be36-ec8efc30e88b">
